### PR TITLE
docs: added images for exclude_current_row in dql/window

### PR DESCRIPTION
**Pull request by michelle-qinqin**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
docs

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**
added images for the example of "exclude_current_row"
improved format for titles of examples in en version
completed translation of exclude_curent_time in en version

### CODE REVIEW

File: docs/en/reference/sql/dql/WINDOW_CLAUSE.md [hunk 1]
```diff
@@ -246,8 +246,8 @@ PARTITION BY col1 ORDER BY col5 ROWS_RANGE BETWEEN 10s PRECEDING AND CURRENT ROW
 ```
 
 
-### **Window Exclude Current Time**
-
+### 2. Window Exclude Current Time
```
Comment (lumianph) [inline docs/en/reference/sql/dql/WINDOW_CLAUSE.md]: this sentence is incorrect, it should be either one as follows
- Window Excluding Current Time (this explains the meaning)
- WINDOW with EXCLUDE CURRENT_ROW (this describes the syntax)

you may help double check other titles

File: docs/zh/reference/sql/dql/WINDOW_CLAUSE.md [hunk 2]
```diff
@@ -188,7 +188,7 @@ WindowUnionClause
         :: = ( 'UNION' TableRefs)
 ```
 
-#### **1.1 Example: Window with union 一张副表**
+**1 Example: Window with union 一张副表**
```
Comment (lumianph) [inline docs/zh/reference/sql/dql/WINDOW_CLAUSE.md]: 这个标题也有点奇怪。。感觉混杂着中英文，如果改一下更加符合中文习惯，比如
基于一张副表的 WINDOW ... UNION

File: docs/zh/reference/sql/dql/WINDOW_CLAUSE.md [hunk 3]
```diff
@@ -197,7 +197,7 @@ WINDOW w1 AS (UNION t2 PARTITION BY col1 ORDER BY col5 ROWS_RANGE BETWEEN 10s PR
 
 ![Figure 2: window union one table](../dql/images/window_union_1_table.png)
 
-#### **1.2 Example: Window with union 多张副表**
+**2 Window with union 多张副表**
```
Comment (lumianph) [inline docs/zh/reference/sql/dql/WINDOW_CLAUSE.md]: 基于多张副表的 WINDOW ... UNION

File: docs/zh/reference/sql/dql/WINDOW_CLAUSE.md [hunk 4]
```diff
@@ -206,7 +206,7 @@ WINDOW w1 AS (UNION t2, t3 PARTITION BY col1 ORDER BY col5 ROWS_RANGE BETWEEN 10
 
 ![Figure 3: window union two tables](../dql/images/window_union_2_table.png)
 
-#### **1.3 Example: Window with union INSTANCE_NOT_IN_WINDOW**
+**3 Window with union INSTANCE_NOT_IN_WINDOW**
```
Comment (lumianph) [inline docs/zh/reference/sql/dql/WINDOW_CLAUSE.md]: 带有 INSTANCE_NOT_IN_WINDOW 的 WINDOW ... UNION

File: docs/zh/reference/sql/dql/WINDOW_CLAUSE.md [hunk 5]
```diff
@@ -217,7 +217,7 @@ WINDOW w1 AS (UNION t2 PARTITION BY col1 ORDER BY col5 ROWS_RANGE BETWEEN 10s PR
 
 ![Figure 4: window union one table with instance_not_in_window](../dql/images/window_union_1_table_instance_not_in_window.png)
 
-#### **1.4 Example: Window with union 列筛选子查询**
+**4 Window with union 列筛选子查询**
```
Comment (lumianph) [inline docs/zh/reference/sql/dql/WINDOW_CLAUSE.md]: 带有列筛选子查询的 WINDOW ... UNION

File: docs/zh/reference/sql/dql/WINDOW_CLAUSE.md [hunk 6]
```diff
@@ -235,16 +235,16 @@ PARTITION BY col1 ORDER BY col5 ROWS_RANGE BETWEEN 10s PRECEDING AND CURRENT ROW
 WindowExcludeCurrentTime 
         ::= 'EXCLUDE' 'CURRENT_TIME'  
 ```
-
-#### **2.1 Example: ROWS 窗口 EXCLUDE CURRENT TIME**
+#### Example
+**1 ROWS 窗口 EXCLUDE CURRENT TIME**
```
Comment (lumianph) [inline docs/zh/reference/sql/dql/WINDOW_CLAUSE.md]: ROWS 类型窗口，带有 EXCLUDE CURRENT_TIME

File: docs/zh/reference/sql/dql/WINDOW_CLAUSE.md [hunk 7]
```diff
@@ -235,16 +235,16 @@ PARTITION BY col1 ORDER BY col5 ROWS_RANGE BETWEEN 10s PRECEDING AND CURRENT ROW
...
 window ROWS, 前1000条到当前条, 除了current row以外窗口内不包含当前时刻的其他数据
```
Comment (lumianph) [inline docs/zh/reference/sql/dql/WINDOW_CLAUSE.md]: 定义了 ROWS 类型窗口，包含前 1,000 行到当前行。除了当前行以外，窗口内不包含当前时刻的其他数据。
（帮助其他地方也类似修饰一下描述）

File: docs/zh/reference/sql/dql/WINDOW_CLAUSE.md [hunk 8]
```diff
@@ -235,16 +235,16 @@ PARTITION BY col1 ORDER BY col5 ROWS_RANGE BETWEEN 10s PRECEDING AND CURRENT ROW
...
 
 ```SQL
 SELECT sum(col2) OVER w1 as w1_col2_sum FROM t1
 WINDOW w1 AS (PARTITION BY col1 ORDER BY col5 ROWS BETWEEN 1000 PRECEDING AND CURRENT ROW EXCLUDE CURRENT_TIME);
 ```
 
-#### **2.2 Example: ROW RANGE 窗口 EXCLUDE CURRENT TIME**
+**2 ROW RANGE 窗口 EXCLUDE CURRENT TIME**
```
Comment (lumianph) [inline docs/zh/reference/sql/dql/WINDOW_CLAUSE.md]: RAWS_RANGE 窗口，带有 EXCLUDE CURRENT_TIME

File: docs/zh/reference/sql/dql/WINDOW_CLAUSE.md [hunk 9]
```diff
@@ -281,7 +282,7 @@ WindowFrameMaxSize
         :: = MAXSIZE NumLiteral
 ```
 
-![Figure 6: window config max size](../dql/images/window_max_size.png)
+![Figure 7: window config max size](../dql/images/window_max_size.png)
 
 #### **Example: ROW RANGE 窗口MAXSIZE**
```
Comment (lumianph) [inline docs/zh/reference/sql/dql/WINDOW_CLAUSE.md]: ROWS RANGE 窗口，带有 MAXSIZE

**Review by lumianph — CHANGES_REQUESTED** [review]
this doc still needs to be further refined

**Comment by lumianph** [comment]
those comments are not addressed yet? (but marked as resolved?) @michelle-qinqin

**Comment by michelle-qinqin** [comment]
> those comments are not addressed yet? (but marked as resolved?) @michelle-qinqin
yes, I closed one comment after I fixed it in local repository, just to avoid missing some comments. I have commit the newest version just now~

**Review by lumianph — APPROVED** [review]

**Pull request merged**